packages/fields/src/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Internal dependencies
+ */
+import { lock } from './lock-unlock';
+
+export const privateApis = {};
+
+lock( privateApis, {} );
```

Comment:
Instead of locking the APIs, I think we should probably make this package a "bundled one" just like the "dataviews" package itself. I know there are two files (configs) to change to make a package a bundled package `tools/webpack/packages.js` and `packages/dependency-extraction-webpack-plugin/lib/util.js`

Reply:
Addressed with [7e04069](https://github.com/WordPress/gutenberg/pull/65230/commits/7e04069cef9dd0740119e8abf16f9d19e58f1bb5)